airbyte-webapp/src/packages/cloud/views/layout/CloudMainView/CloudBottomItems.tsx

```diff
@@ -0,0 +1,52 @@
+import { FormattedMessage } from "react-intl";
+
+import { CreditsIcon } from "components/icons/CreditsIcon";
+
+import { FeatureItem, useFeature } from "hooks/services/Feature";
+import { CloudRoutes } from "packages/cloud/cloudRoutePaths";
+import { useGetCloudWorkspace } from "packages/cloud/services/workspaces/CloudWorkspacesService";
+import { LOW_BALANCE_CREDIT_THRESHOLD } from "packages/cloud/views/credits/CreditsPage/components/LowCreditBalanceHint/LowCreditBalanceHint";
+import { RoutePaths } from "pages/routePaths";
+import { useCurrentWorkspace } from "services/workspaces/WorkspacesService";
+import { MenuContent } from "views/layout/SideBar/components/MenuContent";
+import { NavItem } from "views/layout/SideBar/components/NavItem";
+import SettingsIcon from "views/layout/SideBar/components/SettingsIcon";
+
+import { CloudResourcesDropdown } from "./CloudResourcesDropdown";
+import { CloudSupportDropdown } from "./CloudSupportDropdown";
+
+export const CloudBottomItems: React.FC = () => {
+  const workspace = useCurrentWorkspace();
+  const cloudWorkspace = useGetCloudWorkspace(workspace.workspaceId);
+  const isAllowUpdateConnectorsEnabled = useFeature(FeatureItem.AllowUpdateConnectors);
+
+  return (
+    <ul>
+      <MenuContent>
```

Comment:
I'm not sure I follow here - what's the benefit of mapping over the children like this? If it's just to avoid typing out 4x `<li>` I'm not sure the indirection is worth it.

Reply:
resolved in https://github.com/airbytehq/airbyte/pull/21227/commits/23f818a2bb790415023f030d343e9bbe14d11546